_posts/2018-06-29-14-04-netcore-7-0.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title : "14.4 OData Web API 7.0 now available"
+description: "7.0 release notes"
+category: "14. 7.x Features "
```

Comment:
Can you double check if the space in `Features "` will impact whether this post will show up? You can compare with the other files.

Reply:
They are same.